### PR TITLE
[platform.delegates] Add 0.2.7 version

### PR DESCRIPTION
**Pull request by FreePhoenix888**
**platform.delegates/0.2.7**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
Hello, @FreePhoenix888. Thank you for your contribution! Do you really need the version 0.2.0? Otherwise, keep only the latest.

**Comment by FreePhoenix888** [comment]
@uilianries I have deleted 0.2.0 version

**Comment by FreePhoenix888** [comment]
@uilianries can I ask you for help? I have checked `all logs` but cannot figure out what is wrong

**Comment by FreePhoenix888** [comment]
How to exclude apple-clang from build?

File: recipes/platform.delegates/all/conanfile.py [hunk 1]
```diff
@@ -30,8 +30,8 @@ def _compilers_minimum_version(self):
         return {
             "gcc": "8",
             "Visual Studio": "16",
-            "clang": "10",
```
Comment (uilianries) [inline recipes/platform.delegates/all/conanfile.py:33]: please, revert it.

**Review by uilianries — COMMENTED** [review]
Use https://github.com/conan-io/conan-center-index/blob/1e58a1e366d67152e2a58826117ddae44b5c1505/recipes/json_dto/all/conanfile.py#L44 as reference to exclude apple-clang

**Comment by FreePhoenix888** [comment]
> @FreePhoenix888 just to confirm, the version 0.1.3 does not work with apple-clang too? Otherwise you can submit a rule to skip only for 0.2.7
> 
> ```
> if self.settings.compiler == "apple-clang" and self.version == "0.2.7":
> ```
> 
> Usually we skip a range of versions, but as it could be fixed in next release, we can update later.

We want our package to work with all compilers. I just want to check if everything works with other compilers

**Review by uilianries — APPROVED** [review]

**Pull request merged**